doc/man3/EVP_EncryptInit.pod

```diff
@@ -1305,7 +1305,7 @@ EVP_DecryptInit_ex2() and EVP_DecryptUpdate() return 1 for success and 0 for fai
 EVP_DecryptFinal_ex() returns 0 if the decrypt failed or 1 for success.
 
 EVP_CipherInit_ex2() and EVP_CipherUpdate() return 1 for success and 0 for failure.
-EVP_CipherFinal_ex() returns 0 for a decryption failure or 1 for success.
+EVP_CipherFinal_ex() returns 0 for an encryption / decryption failure or 1 for success.
```

Comment:
(minor) We have two places in the text that say `encryption/decryption`, and one that says `encryption / decryption`. Could we aim to reduce inconsistency rather than increase it? :)

Reply:
Good point. I didn't notice the other two.